benefits/static/css/styles.css

```diff
@@ -557,6 +558,10 @@ footer .footer-links a {
 }
 
 @media (max-width: 992px) {
+  .container.content .btn-lg {
+    padding: 1.1875rem 0.813rem;
```

Comment:
Since the font-size and this padding are `rem`-based, if the user changes their browser's default font size, the button will grow larger than 72px. Is this ok?

| 16px is default font size | 20px is default font size |
| -------------------------- | -------------------------- |
|![16px-default-font](https://user-images.githubusercontent.com/25497886/172457297-cf4f8ab9-20b6-41af-9e34-f6db5559d137.png) | ![20-px-default-font](https://user-images.githubusercontent.com/25497886/172457353-8ad7b72b-82c5-47c7-848e-36d394c264b2.png) |

Reply:
Yeah - We want to respect people's browser zoom/font size settings.